apps/admin-server/src/pages/projects/[project]/statuses/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { PageLayout} from '@/components/ui/page-layout'
+import { Button } from '@/components/ui/button';
+import Link from 'next/link';
+import { ChevronRight, Plus } from 'lucide-react';
+import React from 'react';
+import { useRouter } from 'next/router';
+import { ListHeading, Paragraph } from '@/components/ui/typography';
+import useStatuses from '@/hooks/use-statuses';
+import { RemoveResourceDialog } from '@/components/dialog-resource-remove';
+import toast from 'react-hot-toast';
+
+export default function ProjectStatuses() {
+  const router = useRouter();
+  const { project } = router.query;
+
+  const { data, isLoading, removeStatus } = useStatuses(project as string);
```

Comment:
Als je onder de useStatuses call een copy en cast doet zoals:

const statuses = (data || []) as Status[] // van die model die je eerder hierboven had gedefinieerd en je gebruikt dat i.p.v. data. Dan kan een editor met typehinting makkelijk zien als er ergens een fout is gemaakt met bijv. een verkeerde naam of iets dergelijks, het geeft ook dan een autocomplete wat fouten voorkomt.